cns/restserver/ipam.go

```diff
@@ -574,5 +617,10 @@ func requestIPConfigHelper(service *HTTPRestService, req cns.IPConfigRequest) (c
 	}
 
 	// return any free IPConfig
-	return service.AssignAnyAvailableIPConfig(podInfo)
+	return service.AssignAvailableIPConfigs(podInfo)
+	// podIpInfo, err := service.AssignAvailableIPConfigs(podInfo)
```

Comment:
remove this code and add a todo indicating code needs to be added to release the IPs from partially successful response